portfolio/src/main/webapp/theoffice/theoffice.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>The Office</title>
+    <link rel="stylesheet" href="theoffice.css" />
+    <link rel="stylesheet" href="/../buttons.css" />
+    <script src="theoffice.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <div>
+        <a href="/../index.html" class="button button-black">Home</a>
+      </div>
```

Comment:
It's probably better to move this button above the `<div id="content">`
So a structure like:
```
<body>
  <div id="navigation">
    Navigation stuff
  </div>
  <div id="content">
    Actual page content
  </div>
</body>
```